lib/modules/userTagger.js

```diff
@@ -595,7 +595,11 @@ addModule('userTagger', {
 			}
 			document.getElementById('userTaggerColor').selectedIndex = 0;
 		}
-		this.userTaggerToolTip.setAttribute('style', 'display: block; top: ' + thisXY.y + 'px; left: ' + thisXY.x + 'px; z-index: 20;');
+		if (thisXY.x + 334 < window.innerWidth) {
```

Comment:
You are definitely allowed to use jQuery; it failed because of the double quotes.
![image](https://cloud.githubusercontent.com/assets/7673145/11640993/c08f5848-9d03-11e5-93cf-4d4007a6c402.png)